src/main/java/org/wildfly/extension/elytron/FilteringKeyStoreService.java

```diff
@@ -57,6 +59,15 @@ public void start(StartContext startContext) throws StartException {
             KeyStore unmodifiable = keyStoreService.getValue();
             KeyStore modifiable = keyStoreService.getModifiableValue();
 
+            if (ROOT_LOGGER.isTraceEnabled()) {
+                ROOT_LOGGER.tracef("starting:%n" +
+                        "    aliasFilter = %s%n" +
```

Comment:
I think we still need to remove the String concatenation here and move this onto a single line - then we can loose the wrapping isTraceEnabled